common/utils/src/main/scala/org/apache/spark/util/IvyTestUtils.scala

```diff
@@ -230,12 +226,11 @@ private[deploy] object IvyTestUtils {
       val inside = deps.map(ivyArtifactWriter).mkString("\n")
       "\n  <dependencies>\n" + inside + "\n  </dependencies>"
     }.getOrElse("")
-    content += "\n</ivy-module>"
```

Comment:
Why do we need to delete this line? It will make the content of `ivy.xml` incomplete...

Reply:
Give a fix: https://github.com/apache/spark/pull/43834